docs/requirements.txt

```diff
@@ -1,5 +1,4 @@
 recommonmark==0.6.0
-Sphinx==3.1.1
 sphinx-rtd-theme==0.4.3
 sphinxemoji==0.1.6
```

Comment:
Shouldn't those two at least be removed too?

Reply:
No, we have removed the sphinx requirement because the ReadTheDocs already have it, but the others we need to install yet.